flocker/acceptance/obsolete/test_containers.py

```diff
@@ -123,6 +123,7 @@ def checked(_):
 
     @flaky(u'FLOC-2488')
     @require_moving_backend
+    @run_test_with(async_runner(timeout=timedelta(minutes=6)))
```

Comment:
:question: Perhaps this fixes the flakiness referenced the decorator above?
 * https://clusterhq.atlassian.net/browse/FLOC-2488 (test timeout)